src/korg/_python_interface.py

```diff
@@ -98,34 +105,96 @@ def __repr__(self) -> str:
 
 @_perfect_jl_shadowing
 def get_APOGEE_DR17_linelist(*, include_water: bool = True) -> Linelist:
-    return Linelist(Korg.get_APOGEE_DR17_linelist(include_water=include_water))
+    return Linelist(Korgjl.get_APOGEE_DR17_linelist(include_water=include_water))
 
 
 @_perfect_jl_shadowing
 def get_GALAH_DR3_linelist() -> Linelist:
-    return Linelist(Korg.get_GALAH_DR3_linelist())
+    return Linelist(Korgjl.get_GALAH_DR3_linelist())
 
 
 @_perfect_jl_shadowing
 def get_GES_linelist(*, include_molecules: bool = True) -> Linelist:
-    return Linelist(Korg.get_GES_linelist(include_molecules=include_molecules))
+    return Linelist(Korgjl.get_GES_linelist(include_molecules=include_molecules))
 
 
 @_perfect_jl_shadowing
 def get_VALD_solar_linelist() -> Linelist:
-    return Linelist(Korg.get_VALD_solar_linelist())
+    return Linelist(Korgjl.get_VALD_solar_linelist())
+
+
+def load_ExoMol_linelist(
+    species: str,
+    states_file: str | bytes | os.PathLike,
+    transitions_file: str | bytes | os.PathLike,
+    lower_wavelength: KFloat,
+    upper_wavelength: KFloat,
+    *,
+    isotopes: Iterable[tuple[int, int]] | None = None,
+    line_strength_cutoff: KFloat = -15,
+    T_line_strength: KFloat = 3500,
+    verbose: bool = True,
+) -> Linelist:
+    """
+    Load a linelist from ExoMol. Returns a `~korg.Linelist`, the same as `~korg.read_linelist`.
+
+    Arguments
+    ----------
+    species
+        the species, i.e. the molecule that the linelist is for
+    states_file
+        the path to the ExoMol states file
+    transitions_file
+        the path to the ExoMol transitions file
+    upper_wavelength
+        the upper limit of the wavelength range to load (Å)
+    lower_wavelength
+        the lower limit of the wavelength range to load (Å)
+
+    Keyword Arguments
+    -----------------
+    isotopes
+        a vector of (atomic number, isotope number) pairs for each atom in the species.
+        If not provided, Korg will assume that the molecule is the most abundant isotopologue.
+    line_strength_cutoff
+        the cutoff for the line strength (default: -15) used to filter the
+        linelist.
+    T_line_strength
+        the temperature (K) at which to evaluate the line strength (default: 3500.0)
+    verbose
+        if `true` (default), will print progress information to the console.
+
+    Note: this differs from the Korg.jl API in that it does not support custom isotopic abundances.
+
+    .. warning::
+
+        This functionality is in beta. It's behavior may change without a major version number bump.
```

Comment:
Let's be more explicit
```suggestion
        This functionality is in beta (we may change the behavior in a minor release).
```